daily_practice/challenges/classes-challenges.test.js

```diff
@@ -0,0 +1,49 @@
+// Using the Coffee Maker Class, write tests and complete the challenges
+class Coffee {
```

Comment:
I'm not sure why you are redefining the class here when you have it above. The test files should import the classes, functions, etc from the other file.